workers/handshake-manager/src/manager/matching/external_engine.rs

```diff
@@ -65,7 +66,12 @@ impl HandshakeExecutor {
             // may match the external order as the first party
             match_res.direction = order.side.opposite().match_direction();
             let settle_res = self
-                .try_settle_external_match(other_order_id, price, match_res, response_topic.clone())
```

Comment:
tiny nit: we can now use `ts_price.as_fixed_point()` when defining `price` above